program/src/distribute_base_rewards.rs

```diff
@@ -27,59 +30,69 @@ pub fn process_distribute_base_rewards(
         return Err(ProgramError::InvalidAccountData);
     }
 
-    if !base_fee_wallet.is_writable {
-        msg!("Base fee wallet is not writable");
-        return Err(ProgramError::InvalidAccountData);
-    }
-
     Config::load(restaking_program.key, restaking_config, false)?;
     Ncn::load(restaking_program.key, ncn, false)?;
-
     NcnConfig::load(program_id, ncn.key, ncn_config, false)?;
-
     BaseRewardRouter::load(program_id, ncn.key, epoch, base_reward_router, true)?;
+    BaseRewardReceiver::load(program_id, base_reward_receiver, ncn.key, epoch, true)?;
+    load_associated_token_account(base_fee_wallet_ata, base_fee_wallet.key, &JITO_SOL_MINT)?;
 
-    let group = BaseFeeGroup::try_from(base_fee_group)?;
-
-    // Check if base fee wallet is correct
```

Comment:
We still need to check if the base_wallet is correct, this is saved in the ncn_config